healvis/simulator.py

```diff
@@ -455,6 +469,7 @@ def setup_uvdata(array_layout=None, telescope_location=None, telescope_name=None
     for (a1, a2) in bls:
         bl_array.append(uvutils.antnums_to_baseline(a1, a2, 1))
     bl_array = np.asarray(bl_array)
+    print("Nbls: {}".format(bl_array.size))
```

Comment:
again, smells like a debug print. If so, but you want to keep it around, you should possibly convert it to a logging call.

Reply:
It is. I've gotten rid of it.